### PR TITLE
Fix persistent damage of cloak of gnawing leaves

### PR DESCRIPTION
**Pull request by rectulo**

### CODE REVIEW

**Comment by stwlam** [comment]
Bleed is always persistent, so there's nothing to fix here.

File: packs/data/equipment.db/cloak-of-gnawing-leaves.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
...
+            "value": "<p>This cloak appears to be woven from a thousand living leaves, hungry for flesh and eager to defend the cloak's wearer.</p>\n<p><strong>Activate</strong> <span class=\"pf2-icon\">r</span> envision</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You are damaged by a melee attack from an adjacent creature</p>\n<hr />\n<p><strong>Effect</strong> The leaves lash out at your attacker, rising up to reveal snapping jaws made of wicked thorns. The triggering creature must attempt a @Check[type:reflex|dc:17] saving throw.</p>\n<p><strong>Success</strong> The creature is unaffected.</p>\n<p><strong>Failure</strong> The creature takes [[/r 1d4[piercing]]].</p>\n<p><strong>Critical Failure</strong> The creature takes [[/r 2d4[piercing]]] and [[/r 1[persistent,bleed]]].</p>"
```
Comment (stwlam) [inline packs/data/equipment.db/cloak-of-gnawing-leaves.json:9]: Are you wanting to change the displayed language here? Something like `The creature takes [[/r {2d4[piercing],1[bleed]}]]{2d4 piercing and 1 persistent bleed}`?

Comment (rectulo) [reply packs/data/equipment.db/cloak-of-gnawing-leaves.json:9]: I understand that bleed damages are always persistent and the smplier, the better for me.

But in all the other files I had translated, the format for bleed damages are not only [[bleed]]
For all the 159 others occurencies, I have [[persistent,bleed]]

image

If bleed is sufficient in the format, could you tell me why the others are not under this format ?

**Comment by rectulo** [comment]
I have [bleed] without persistent first inside 2 others new description for items of the last AP
- Cloak of Thirsty Fronds
- Cloak of Devouring Thorns

File: packs/data/equipment.db/cloak-of-gnawing-leaves.json (hunk 1 above)
Comment (TikaelSol) [reply packs/data/equipment.db/cloak-of-gnawing-leaves.json:9]: Likely just got migrated that way, we can fix it.

Comment (rectulo) [reply packs/data/equipment.db/cloak-of-gnawing-leaves.json:9]: Bad catch for a good fix if I understand well.

**Comment by TikaelSol** [comment]
I fixed the other outliers, so this one can be closed as everything is no in the same format.

**Pull request closed without merging**